install_config/configuring_pipeline_execution.adoc

```diff
@@ -67,10 +67,17 @@ jenkinsPipelineConfig:
 
 [IMPORTANT]
 ====
-When a Pipeline build configuration is created,
-{product-title} instantiates the `*jenkinsPipelineConfig*` template *only if*
-no existing service name in the project matches the `*serviceName*` field.
-This means `*serviceName*` must be chosen such that it is unique in the project.
+When a Pipeline build configuration is created, {product-title} instantiates
+the `*jenkinsPipelineConfig*` template *only if* no existing service name in
+the project matches the `*serviceName*` field.  This means `*serviceName*` must
+be chosen such that it is unique in the project.
+====
+
+[IMPORTANT]
+====
+If you're using an external Jenkins server to manage your pipeline, you'll need
+to create a service for {product-title} to use it.
+xref:../dev_guide/integrating_external_services.adoc#dev-guide-integrating-external-services[Learn more about integrating external services.]
```

Comment:
I'm going to absorb the warning above to try not to repeat information, how does that sound?